conjure-java-jackson-serialization/build.gradle

```diff
@@ -8,8 +8,12 @@ dependencies {
     compile "com.fasterxml.jackson.datatype:jackson-datatype-jsr310"
     compile "com.fasterxml.jackson.dataformat:jackson-dataformat-cbor"
 
+    implementation 'com.palantir.conjure.java:conjure-lib', {
```

Comment:
The SafeLong serializer needs to live in this project, rather than with SafeLong to avoid wire breaks from mixed conjure-java/conjure-java-runtime versions.